[ADD] awesome_clicker: create a clicker game (OWL tutorial)

robal-odoo opened the pull request.

robal-odoo committed:
[ADD] awesome_clicker: add clicker in systray

robal-odoo committed:
[IMP] awesome_clicker: count external clicks

robal-odoo committed:
[IMP] awesome_clicker: add clicker action

robal-odoo committed:
[IMP] awesome_clicker: move the clicker state to a service

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/tutorials/pull/1090.png)](https://mergebot.odoo.com/odoo/tutorials/pull/1090)

robal-odoo committed:
[IMP] awesome_clicker: add a "useClicker" hook

robal-odoo committed:
[IMP] awesome_clicker: humanize display of click numbers

robal-odoo committed:
[IMP] awesome_clicker: add tooltip to ClickValue component

robal-odoo committed:
[IMP] awesome_clicker: add clickbots

- Add clickbots
- Move event registration for clicks into the clicker_service + set useCapture to true

robal-odoo committed:
[REF] awesome_clicker: refactor clicker state to a class model

robal-odoo committed:
[IMP] awesome_clicker: add milestone effect

robal-odoo committed:
[IMP] awesome_clicker: add bigbots

robal-odoo committed:
[IMP] awesome_clicker: add power level increases

robal-odoo committed:
[REF] awesome_clicker: big refactoring

- Extract game data such as shop items and milestones to a separate file for game data
- Make the shop/action template more generic

robal-odoo committed:
[IMP] awesome_clicker: add random rewards

robal-odoo committed:
[IMP] awesome_clicker: add random reward on patched form controller

robal-odoo committed:
[IMP] awesome_clicker: add commands

robal-odoo committed:
[IMP] awesome_clicker: add fruits and trees

robal-odoo committed:
[IMP] awesome_clicker: convert systray to dropdown

robal-odoo committed:
[IMP] awesome_clicker: display shop/action using notebook components

robal-odoo committed:
[IMP] awesome_clicker: add state persistence
